nim-rx.el

```diff
@@ -100,30 +99,22 @@ This variant of `rx' supports common nim named REGEXPS."
       (cond ((null regexps)
              (error "No regexp"))
             ((cdr regexps)
-             (rx-to-string `(and ,@regexps) t))
+             (rx-to-string (cons 'and regexps) t))
             (t
              (rx-to-string (car regexps) t)))))
 
-  ;; based on nim manual
   (add-to-list 'nim-rx-constituents
-               ;; note rx.el already has ‘letter’
```

Comment:
I don't think we should delete this. Nim has there spec for the identifier (https://nim-lang.org/docs/manual.html#lexical-analysis-identifiers-keywords)

Reply:
I actually read this and the follow up part: 

> Currently any Unicode character with an ordinal value > 127 (non ASCII) is classified as a letter and may thus be part of an identifier but later versions of the language may assign some Unicode characters to belong to the operator characters instead.

To be honest it is much more likely that any unicode character that is classified as a letter by unicode will be classified as a letter by the Nim compiler.